docs/guide/docs-as-code.rst

```diff
@@ -53,3 +57,19 @@ and are gaining adoption in the writing community.
 .. _panel: https://www.youtube.com/watch?v=Y2TGwUPb8R4
 .. _Margaret Eker and Jennifer Roundeau: https://www.youtube.com/watch?v=JvRd7MmAxPw
 .. _Rachel Whitten: https://www.youtube.com/watch?v=dHdBsNxtKeI
+
+*Docs as Code* at other Conferences, Video Casts and Articles
+---------------------------------------------------
+
+The approach has been presented at several other conferences or just video casts by different speakers.
+
+**2017**
+
+* Software Architecture Summit 2017, Berlin, Germany: Gernot Starke, Ralf D. Müller: https://software-architecture-summit.de/softwarearchitektur/hitchhikers-guide-to-architecture-documentation/ (Half-Day-Workshop)
+
+**2018**
+
+* I'd rather be writing: Tom Johnson: https://www.youtube.com/watch?v=Z3e_38WS-2Q
```

Comment:
Would you mind linkifying these?

Reply:
yes, sure. had to lookup the rst syntax, but I guess now it is fine, isn't it?